e2e/_suites/helm/helm_charts_test.go

```diff
@@ -362,7 +362,7 @@ func (ts *HelmChartTestSuite) getResourceName(resource string) string {
 	} else if resource == k8s.ResourceTypes.ClusterRoleBinding {
 		return strings.ToLower(ts.Name + "-" + ts.Name + "-cluster-role-binding")
 	} else if resource == k8s.ResourceTypes.ConfigMap {
-		if ts.Name == "metricbeat" {
+		if ts.Name == "filebeat" || ts.Name == "metricbeat" {
```

Comment:
@jmlrt I checked that filebeat is now deployed as daemonset. With this change now the tests pass